src/test/resources/pacts/publicapi-connector-search-refunds-with-page-and-display.json

```diff
@@ -0,0 +1,149 @@
+{
+  "consumer": {
+    "name": "publicapi"
+  },
+  "provider": {
+    "name": "connector"
+  },
+  "interactions": [
+    {
+      "description": "Return all refunds when refunds exist",
+      "providerStates": [
+        {
+          "name": "Account exists and has 2 refunds"
```

Comment:
you probably want to add a "params" block here with `"gateway_account_id": 777`. see https://github.com/alphagov/pay-publicapi/blob/75366929a6b6868b27ac349cc3408e03ae7e67a8/src/test/resources/pacts/publicapi-connector-get-payment-with-delayed-capture-true.json#L14 as an example.